packages/orchestrator/main.go

```diff
@@ -246,12 +247,24 @@ func run(port, proxyPort uint) (success bool) {
 		zap.L().Fatal("failed to create feature flags client", zap.Error(err))
 	}
 
+	var clickhouseClient clickhouse.Clickhouse
+	clickhouseConnectionString := os.Getenv("CLICKHOUSE_CONNECTION_STRING")
```

Comment:
can we use feature flags for turning collecting events on and off if something goes wrong?